test_bundle.py

```
@@ -0,0 +1,70 @@
+
+# ─ Module: mypackage.formatter ─
+"""Data formatting module."""
+
+
+def format_data(data):
+    """Format the processed data."""
+    return f"Formatted: {data}"
+
+
+# ─ Module: mypackage.config ─
+import types
```

Pylint (3.3.7)

[convention] Import "import types" should be placed at the top of the module

(C0413)